vocode/streaming/models/endpoint_classifier_model.py

```diff
@@ -0,0 +1,37 @@
+import string
+import torch.nn.functional as F
+import time
+import torch
+from transformers import DistilBertTokenizerFast, DistilBertForSequenceClassification
+
+class EndpointClassifier:
```

Comment:
the `/models` directory is about data models, not ML models - mind moving this to `streaming/transcriber`?